substrate/client/rpc-spec-v2/src/sudo_session_keys/tests.rs

```diff
@@ -0,0 +1,66 @@
+// This file is part of Substrate.
+
+// Copyright (C) Parity Technologies (UK) Ltd.
+// SPDX-License-Identifier: GPL-3.0-or-later WITH Classpath-exception-2.0
+
+// This program is free software: you can redistribute it and/or modify
+// it under the terms of the GNU General Public License as published by
+// the Free Software Foundation, either version 3 of the License, or
+// (at your option) any later version.
+
+// This program is distributed in the hope that it will be useful,
+// but WITHOUT ANY WARRANTY; without even the implied warranty of
+// MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the
+// GNU General Public License for more details.
+
+// You should have received a copy of the GNU General Public License
+// along with this program. If not, see <https://www.gnu.org/licenses/>.
+
+use super::{sudo_session_keys::SudoSessionKeys, MethodResult};
+use codec::Decode;
+use jsonrpsee::{types::EmptyServerParams as EmptyParams, RpcModule};
+
+use crate::sudo_session_keys::api::SudoSessionKeysServer;
+use sp_core::{
+	crypto::ByteArray,
+	testing::{ED25519, SR25519},
+};
+use sp_keystore::{testing::MemoryKeystore, Keystore};
+use std::sync::Arc;
+use substrate_test_runtime_client::{
+	self,
+	runtime::{Block, SessionKeys},
+	Backend, Client, DefaultTestClientBuilderExt, TestClientBuilderExt,
+};
+
+fn setup_api() -> (Arc<MemoryKeystore>, RpcModule<SudoSessionKeys<Client<Backend>, Block>>) {
+	let keystore = Arc::new(MemoryKeystore::new());
+	let client = Arc::new(substrate_test_runtime_client::TestClientBuilder::new().build());
+	let api = SudoSessionKeys::new(client, keystore.clone()).into_rpc();
+
+	(keystore, api)
+}
+
+#[tokio::test]
+async fn sudo_session_keys_unstable_generate() {
```

Comment:
add a test for the MethodResult::Err as well? 

Or just that serialize works...

Reply:
Tested in the main crate as part of some other pr, thanks!